pkg/update/deps/cleanup.go

```diff
@@ -40,13 +40,18 @@ func gitCheckout(p *config.Pipeline, dir string, mutations map[string]string) er
 		return err
 	}
 
+	gitAuth, err := wgit.GetGitAuth(repoValue)
```

Comment:
same comment here - do we need to authenticate this request?

Reply:
I think not, @hectorj2f can you confirm and I can remove it in this PR

Reply:
I've made this change, will keep the conversation open for when @hectorj2f can agree

Reply:
So we need that git credentials to be able to checkout private repos such as the enterprise-packages.

Reply:
I was looking from my phone. I talked with @rawlingsj. This can go away now.